app/controllers/api/v1/admin/authentication_admin_controller.rb

```diff
@@ -4,6 +4,7 @@ module Api
   module V1
     module Admin
       class AuthenticationAdminController < Api::V1::ApiController
+        skip_before_action :authenticate_api_v1_user!
```

Comment:
Seeing as the Admin has its own authentication, we could probably skip this for all the Admin controllers directly, what do you think?